### ISSUE TITLE
add the capacity to create categorical or opaque (length-only) histograms through the react ui

### ISSUE THREAD

**Issue by jbr** [divviup/divviup-api#438]

**Comment by divergentdave** [comment]
I created a task with divviup-api 0.0.22, and the task creation form had a field labeled "Selects the number of histogram buckets or counters". The client configuration snippet on the resulting task details pages includes `"buckets": [8]`. Playing around with the field again, I see it accepts (and reformats) comma-separated lists. I think this just means the copy is a bit ahead of the changes to be implemented in this issue for the time being.

**Comment by jbr** [comment]
Yeah that’s been out of date for a bit; I’m not entirely sure when that copy change happened but we don’t support anything other than continuous integer named buckets currently
